lib/assessment.js

```diff
@@ -24,9 +41,16 @@ var sum = function sum() {
 // if invoking the callback for any element
 // in the array returns a truthy value
 var any = function any(array, callback, thisArg) {
+  var.result = true;
+  array.forEach(function(currentValue, index, array) {
+
+    result = result && callback(currentValue, index, array);
```

Comment:
Very close! One issue. If we use `&&`, our final result will only be true if _all_ elements satisfy the condition.

Let's work through an example. Say that a hypothetical callback function `isEven` returns `true` for even numbers and `false` for odd numbers. 
Starting at the left of the array `[0,1,2,4,6]`, we would first use the callback to compare `true` and `isEven(0)`; `isEven(0)` is true, so the result (`true && true`) gives the result `true`. We then take that first result, `true`, and compare it with `isEven(1)`. However, `isEven(1)` is _false_, so performing  `true && false` gives a **false** result. This `false` then gets compared with `isEven(2)`, and although `isEven(2)` is `true`, `false && true` still returns `false`. 

In other words, what you're doing is
```javascript
((((true) && isEven(0)) && isEven(1)) && isEven(2)) ....
```
which boils down to
```javascript
true && isEven(0) && isEven(1) && isEven(2) ...
```
This expression will only be true if **all** are true.

However, what we want is to see if **any** are true. To do that, you want to use an 'or' (`||`). 
```javascript
array.forEach(function(previous, current){
  return previous || callback(current);
});
```